README.md

```diff
@@ -59,17 +36,54 @@ Doctype interface exported by the [`@ceramicnetwork/ceramic-common` library](htt
 
 `Resolver` instance exported by the [`did-resolver` library](https://github.com/decentralized-identity/did-resolver)
 
-#### Accessors
+### ResolverOptions
+
+`ResolverOptions` interface exported by the [`dids` library](https://github.com/ceramicnetwork/js-did)
+
+### DocID
+
+The ID of a Ceramic document.
+
+```ts
+type DocID = string
+```
+
+### Definition
 
 ```ts
-interface Accessors {
-  list: (id?: string) => Promise<Array<string>>
-  get: <T = any>(idOrKey: string, key?: string) => Promise<T>
-  set: <T = any>(key: string, value: T) => Promise<void>
-  remove: (key: string) => Promise<void>
+interface Definition<T extends Record<string, unknown> = Record<string, unknown>> {
+  name: string
+  schema: DocID
+  description?: string
+  url?: string
+  config?: T
 }
 ```
 
+### DefinitionsAliases
+
+```ts
+type DefinitionsAliases = Record<string, DocID>
+```
+
+### SchemaType
+
+```ts
+type SchemaType =
+  | 'BasicProfile'
+  | 'Definition'
+  | 'DocIdDocIdMap'
+  | 'DocIdList'
+  | 'DocIdMap'
+  | 'StringMap'
+```
+
+### SchemasAliases
+
+```ts
+type SchemasAliases = Record<SchemaType, DocID>
+```
```

Comment:
What's a schema alias?

Reply:
A mapping of defined schema name to doc ID, for example `DocIdMap`